book/src/config.md

```diff
@@ -140,9 +140,7 @@ For top-level crates, defaults to `safe-to-run`.
 
 #### `dependency-criteria`
 
-Allows overriding the above values on a per-dependency basis. Similar in format
-to the [equivalent field](audit-entries.md#dependency-criteria) in audit
-entries.
+Allows overriding the above values on a per-dependency basis.
```

Comment:
As @hpenne mentioned, it'd be nice to move some of the examples from the `audit-entries.md` file.

It's probably worth also updating the wording for the description of the policy table to reflect that policy fields are respected for both first-party and third-party/audit-as-crates-io crates (currently it claims that they're only respected for first-party crates). IIRC currently we only respect `criteria` and `dev-criteria`, but we will respect `dependency-criteria` for third-party/audit-as-crates-io crates.